ansible/playbook/README.md

```diff
@@ -106,14 +108,6 @@ More information on these documents:
 
 This system already includes a static inventory.
```

Comment:
Can you add the link to the resource on github please ? @jacobdotcosta

Reply:
done.